README.md

```diff
@@ -66,9 +66,9 @@ If there are changes made in the Original repository, you can sync the fork to k
 4. ```git merge upstream/master``` to sync your local `master` branch with `upstream/master`. **Note**: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
 ## Requirements
-* [ArcGIS Runtime SDK for .NET 100.8 or higher](https://developers.arcgis.com/net/latest/)
+* [ArcGIS Runtime SDK for .NET 100.9 or higher](https://developers.arcgis.com/net/latest/)
 * [XCode 11 or higher](https://developer.apple.com/xcode/downloads/)
-    * App supports iOS 12 and above, but requires the iOS 13 SDK to build
+    * App supports iOS 12 and above, but requires the iOS 13 SDK to build. Because ArcGIS Runtime uses [Metal](https://developer.apple.com/metal/), you will not be able to test the app on iOS 12 simulator.
```

Comment:
Adding this requirement stating only that the app won't simulate using iOS 12 simulator is fine, though if you want to keep it this way I would add another line that mentions the min macOS requirement is Catalina. This is because Mojave won't run metal in any iOS simulator.